assets/_scss/components/_forms.scss

```diff
@@ -23,7 +23,7 @@ form {
     margin-bottom: 1.5em;
   }
 
-  input[name="password"] {
+  input[name="password"], input[name="confirmPassword"] {
```

Comment:
@msecret would you want `[name="password", [name="confirmPassword"]` nested inside `form`?